packages/client/wallets/aa/src/utils/blockchain.ts

```diff
@@ -51,3 +62,15 @@ export async function convertData(
     if (resolvedData === undefined) return undefined;
     return hexlify(resolvedData) as `0x${string}`;
 }
+
+export async function verifyMessage({ address, message, signature, chain }: VerifyMessageParams) {
+    const publicClient = createPublicClient({
+        transport: http(getBundlerRPC(chain)),
```

Comment:
Does chain matter? I think signatures are the same on all chain, so if we can confirm, I'd like to hard-code this to one EVM chain (maybe Ethereum) so that the client doesn't have to think about the variable